### DISCUSSION TITLE
Add on-demand static import

### DISCUSSION THREAD

**Discussion by Overhear8805** [mfussenegger/nvim-jdtls#341] [Q&A] [answered]
I'm finally getting my hands dirty in migrating from IntelliJ towards neovim for Java development. One thing I use often in my daily work is extracting static classes to static imports, e.g. these kind of imports `import static org.joda.time.DateTime.now;`

So instead of having to call `DateTime.now()` you just call `now()` in the given class you're in.

My question is if it's possible to make that extraction a code action? I am fairly sure that it's implemented in eclipse.jdt.ls given this issue (https://github.com/eclipse/eclipse.jdt.ls/issues/1203) but I suspect that what is missing is bindings between nvim-jdtls and the Eclipse language server? 
I'm happy to contribute it myself but I would probably need someone to point me in the right direction.

**Comment by mfussenegger** [comment, marked as answer]
It's possible but requires configuration, see https://github.com/mfussenegger/nvim-jdtls/issues/302#issuecomment-1206131439
